test/show.jl

```diff
@@ -64,3 +63,46 @@ end
      t0: 0
      b: 0"""
 end
+
+@testset "KeplerianOrbit" begin
+    orbit = KeplerianOrbit(
+        ρ_star = 1.0,
+        R_star = cbrt(3.0/(4.0*π)),
+        period = 2.0,
+        ecc = 0.0,
+        t_0 = 0.0,
+        incl = π / 2.0,
+        Ω = 0.0,
+        ω = 0.0,
+    )
+
+    @test sprint(show, "text/plain", orbit) === """
```

Comment:
```suggestion
    @test repr("text/plain", orbit) === """
```

Reply:
f0e478a f7ed0c4

I also added the preferred units to the test harness, similarly to [here](https://github.com/JuliaAstro/Transits.jl/pull/23#commitcomment-58931035)